src/components/steps/DecryptXprv.vue

```diff
@@ -99,8 +100,11 @@ export default {
         backupPassword: this.password,
       })
       if (!this.xprvError) {
-        await this.setXprv({ result: this.fileInfo.data.master_key })
-        await this.setBackupPassword({ result: this.password })
+        // FIXME: unify calls and move logic into vuex
```

Comment:
I'm requesting a change here just for the sake of preventing a PR from merging with a FIXME in it :wink: